docker/windows/run.ps1

```diff
@@ -0,0 +1,9 @@
+if ((Get-NetFirewallRule -DisplayName "Docker Proxy" -ErrorAction Ignore) -eq $null)
```

Comment:
Please add a comment to this file that they need to enable the expose daemon setting in Docker.

Reply:
Ok